dynamic_math.js

```diff
@@ -1,7 +1,8 @@
 function render_preview() {
     $(".js-preview-tab").on("click", function(e) {
         function didLoadPreview() {
-            if (!$(".js-preview-tab").hasClass('selected')) {
+            let tab = $(".js-preview-tab");
```

Comment:
Not sure if `var` would be more appropriate given that MathJax tries to target IE6?